packages/bpk-component-content-cards/README.md

```diff
@@ -0,0 +1,38 @@
+# BpkContentCards
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-content-cards --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkContentCards from 'bpk-component-content-cards';
+
+<BpkContentCards
+    headerText='Header'
+    cards=[
+        {
+        imageLink: 'http://skyscanner.net/card-1-image-link.jpg',
+        imageAlt: '',
+        headline: 'Card headline',
+        description: 'Card description',
+        ctaLink: 'http://skyscanner.net/card-1-cta-link.html',
```

Comment:
Just thinking out loud, maybe this API could be improved, now that we decouple it from the context of Sponsored Destination Explorer. E.g. `imageLink` can be interpreted as "where does the image link to" and not "image to render src/href". In a similar way, `ctaLink` might be something different, as it suggests there's a Call To Action (CTA), but actually whole card is a link to some content.

Examples (just some quick thought):
```suggestion
        image: { url: 'http://skyscanner.net/card-1-image-link.jpg', alt: 'White sand beach in a shiny weather of Australian coast', },
        headline: 'Card headline',
        description: 'Card description',
        link: 'http://skyscanner.net/example.html', // or href/url/keep ctaLink
```

I am fine with current API, just thinking if we can improve it right now.

Reply:
<img width="817" alt="Screenshot 2022-08-11 at 17 09 58" src="https://user-images.githubusercontent.com/58072716/184166851-fe3c3450-12c0-4e19-83ff-d5e3e8969f01.png">
this creates a problem with the eslint as it thinks that we're passing the word 'image' as the alt and disabling the check for that line doesn't work

Reply:
I changed it from image to img and it fixes the problem

Reply:
you don't have to make any changes here right now, I'm raising it to open a discussion, maybe we can involve other people in this